### PR TITLE
Buffs update

### PR DESCRIPTION
**Pull request by Linkaaaaa**
Created a new class BuffImages to store the url strings and alleviate repetitions within the Buff class while easing readability.

Buffs Added:
- 2 Ascended foods (+ 20 IDs missing fixed in previous PR)
- 89 Potions
- 48 Fight Specific (Achievement Eligibility)
- 325 Foods (4 included in previous PR)
- 12 Sigil effects
- 4 Rune effects
- Updated Bolt Break & Power Share icons

Found missing IDs for the following Enhancements:
- 3686 (EVTC Inspector: "Do Nothing Transformation Buff")
- 61610 (EVTC Inspector: "Transformation")
- 61339

### CODE REVIEW

File: GW2EIEvtcParser/EIData/Buffs/Buff.cs [hunk 1]
```diff
@@ -324,6 +324,7 @@ public bool Available(CombatData combatData)
             new Buff("Superior Rune of the Monk", SuperiorRuneOfTheMonk, Source.Gear, BuffStackType.Stacking, 10, BuffClassification.Gear, BuffImages.SuperiorRuneOfTheMonk, 93543, GW2Builds.EndOfLife),
             new Buff("Superior Rune of the Cavalier", SuperiorRuneOfTheCavalier, Source.Gear, BuffStackType.CappedDuration, 5, BuffClassification.Gear, BuffImages.SuperiorRuneOfTheCavalier),
             new Buff("Healing Glyph (Druid Runes)", HealingGlyph, Source.Gear, BuffStackType.CappedDuration, 2, BuffClassification.Gear, BuffImages.ConsumeRation),
+            new Buff("Critical (Daredevil Runes)", Critical, Source.Gear, BuffStackType.CappedDuration, 4, BuffClassification.Gear, BuffImages.Critical),
```
Comment (EliphasNUIT) [inline GW2EIEvtcParser/EIData/Buffs/Buff.cs:327]: BuffStackType.CappedDuration ?

Comment (EliphasNUIT) [reply GW2EIEvtcParser/EIData/Buffs/Buff.cs:327]: The id you provide links to a buff with BuffStackType.Stacking and a capacity of 25, did you get the wrong id?

File: GW2EIEvtcParser/EIData/Buffs/Buff.cs [hunk 2]
```diff
@@ -322,6 +322,7 @@ public bool Available(CombatData combatData)
             new Buff("Superior Sigil of Doom", SigilOfDoom, Source.Gear, BuffClassification.Gear, BuffImages.SuperiorSigilOfDoom),
             // Runes
             new Buff("Superior Rune of the Monk", SuperiorRuneOfTheMonk, Source.Gear, BuffStackType.Stacking, 10, BuffClassification.Gear, BuffImages.SuperiorRuneOfTheMonk, 93543, GW2Builds.EndOfLife),
+            new Buff("Superior Rune of the Cavalier", SuperiorRuneOfTheCavalier, Source.Gear, BuffStackType.CappedDuration, 5, BuffClassification.Gear, BuffImages.SuperiorRuneOfTheCavalier),
```
Comment (EliphasNUIT) [inline GW2EIEvtcParser/EIData/Buffs/Buff.cs:325]: You can remove "BuffStackType.CappedDuration, 5", that is a force override stacking type

File: GW2EIEvtcParser/EIData/Buffs/Buff.cs (hunk 1 above)
Comment (Linkaaaaa) [reply GW2EIEvtcParser/EIData/Buffs/Buff.cs:327]: The rune gives this effect https://wiki.guildwars2.com/wiki/Critical_(effect) even though it's not specified on the wiki, whenever you dodge you lose the current stack and gain a new one, have i missinterpreted the behavior?

File: GW2EIEvtcParser/EIData/Buffs/Buff.cs [hunk 3]
```diff
@@ -197,845 +198,1322 @@ public bool Available(CombatData combatData)
...
+            new Buff("Superior Sigil of the Stars", SuperiorSigilOfTheStars, Source.Gear, BuffStackType.Stacking, 25, BuffClassification.Gear, BuffImages.SuperiorSigilOfStars),
+            new Buff("Superior Sigil of Severance", SuperiorSigilOfSeverance, Source.Gear, BuffClassification.Gear, BuffImages.SuperiorSigilOfSeverance),
+            new Buff("Superior Sigil of Doom", SigilOfDoom, Source.Gear, BuffClassification.Gear, BuffImages.SuperiorSigilOfDoom),
+            // Runes
+            new Buff("Superior Rune of the Monk", SuperiorRuneOfTheMonk, Source.Gear, BuffStackType.Stacking, 10, BuffClassification.Gear, BuffImages.SuperiorRuneOfTheMonk, 93543, GW2Builds.EndOfLife),
+            new Buff("Superior Rune of the Cavalier", SuperiorRuneOfTheCavalier, Source.Gear, BuffStackType.CappedDuration, 5, BuffClassification.Gear, BuffImages.SuperiorRuneOfTheCavalier),
+            new Buff("Healing Glyph (Druid Runes)", HealingGlyph, Source.Gear, BuffStackType.CappedDuration, 2, BuffClassification.Gear, BuffImages.ConsumeRation),
```
Comment (EliphasNUIT) [inline GW2EIEvtcParser/EIData/Buffs/Buff.cs:326]: Same thing here.

**Comment by EliphasNUIT** [comment]
Thanks!

**Pull request merged**